gdpr/impl.go

```diff
@@ -223,6 +223,16 @@ func (p *permissionsImpl) parseVendor(ctx context.Context, vendorID uint16, cons
 	return
 }
 
+// DisallowHostCookies represents a GDPR permissions policy with host cookie syncing disallowed
+type DisallowHostCookies struct {
+	*permissionsImpl
+}
+
+// HostCookiesAllowed always returns false
+func (p *DisallowHostCookies) HostCookiesAllowed(ctx context.Context, consent string) (bool, error) {
+	return false, nil
```

Comment:
Re-reading the spec, I think this should return true. We want to disable the host cookie check if the host gvl id is not configured. Whereas before, the GVL code would check for consent given to host 0 and always return false.